python/ray/_private/thirdparty/pynvml/__init__.py

```diff
@@ -1,4 +1,4 @@
 from ray._private.thirdparty.pynvml.pynvml import *
 # nvdia-ml-py version
 # Note: we pick this version to use the V2 API which is supported by older drivers
```

Comment:
This line can be deleted since with the upgrade, we are using NVML V3 API now which supports drivers released after 2012.